README.md

```diff
@@ -65,7 +65,7 @@ You may also want to check out [Meilisearch 101](https://www.meilisearch.com/doc
 
 ## ⚡ Supercharge your Meilisearch experience
 
-Say goodbye to server deployment and manual updates with [Meilisearch Cloud](https://www.meilisearch.com/pricing?utm_campaign=oss&utm_source=engine&utm_medium=meilisearch). Get started with a 14-day free trial! No credit card required.
+Say goodbye to server deployment and manual updates with [Meilisearch Cloud](https://www.meilisearch.com/cloud?utm_campaign=oss&utm_source=github&utm_medium=meilisearch). Get started with a 14-day free trial! No credit card required.
```

Comment:
Hello @macraig 
thanks for the PR. What is the new convention? All the other link in the integration README have `utm_source=integration`. Why the engine repo should use `github` instead of `engine`?

Reply:
I think it's finally right :soblol: thanks for the catch!